src/ui/scripts/cookies.js

```diff
@@ -1,10 +1,11 @@
-var hideButton = document.querySelector('.js-hide');
-var cookieBanner = document.querySelector('.govuk-cookie-banner');
-var acceptButton = document.querySelector('.js-cookies-button-accept');
-var rejectButton = document.querySelector('.js-cookies-button-reject');
-var changeCookiesSubmitButton = document.querySelector('.change-cookies-submit-button');
+const hideButton = document.querySelector('.js-hide');
+const cookieBanner = document.querySelector('.govuk-cookie-banner');
+const acceptButton = document.querySelector('.js-cookies-button-accept');
+const rejectButton = document.querySelector('.js-cookies-button-reject');
+const changeCookiesSubmitButton = document.querySelector('.change-cookies-submit-button');
+const cookieName = '__Secure-optionalCookies';
 
-var expiryDate = function () {
+const expiryDate = function () {
```

Comment:
Can you run `npm run build` in the UI please, scripts need to be compiled for browser consumption

Might not work for older browsers when changing from `var` to `const`, if our webpack config doesn't handle this

Reply:
Have ran and pushed the changes.